docs/features/http-methods-webhooks.md

```diff
@@ -139,34 +158,18 @@ HTTP Methods Deployed:
   - HttpServer.handleSimplePlainRequest: https://<lambdaUrl>/HttpServer/handleSimplePlainRequest
 
 App Dashboard URL: https://app.genez.io/project/<projectId>/<projectEnvId>
-```
+````
 
 ## HTTP types
 
-### GenezioHttpRequest
-
-#### Properties
-
-- **headers - required:** A dictionary that contains the headers.
-- **http - required:** An object that has the following properties:
-  - **method:** The HTTP method.
-  - **path:** The path of the request.
-  - **protocol:** The HTTP version used.
-  - **userAgent:** The request's user agent.
-  - **sourceIp:** The IP of the source.
-- **queryStringParameters - optional:** A dictionary that contains the query parameters.
-- **timeEpoch - required:** Timestamp when the request was made.
-- **rawBody - required:** A string with the unparsed body
-- **body - required:** An object that represents the request's body. If the value is JSON, the value of this variable is a JSON object. If the value is binary, the value of this variable is a Buffer. If the value is text, the value of this variable is also text.&#x20;
-
-### GenezioHttpResponse
+<Tabs>
+  <TabItem value="decorators" label="TS/JS">
+    If you are using TypeScript or JavaScript, you need to use the standard `Request` and `Response` objects provided by the `@types/node` library.
 
-#### Properties
+    To learn more about these types, you can check out the Fetch API documentation for the [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) and [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) objects.
 
-- **body - required:** An object that represents the response's body. The type of this variable can be Object, String, or Buffer.
-- **headers - optional:** A dictionary that contains the headers.
-- **statusCode - required:** The status code of the response.
-- **isBase64Encoded - optional:** This flag can be set to `true` or `false` to indicate if the `body` is base64 encoded. This flag is optional and can be omitted.&#x20;
+  </TabItem>
+</Tabs>
```

Comment:
Isn't this section duplicated with the `info` Admonition above? I would remove it entirely.